exercises/matrix/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "matrix",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
@sshine was requesting 1.3.0 in #1577. Is there a strong argument for 1.2.1?

Reply:
Bumped to 1.3.0; I don't think there's an argument for either, but the minor version bump is fine.

Reply:
[Test data versioning](https://github.com/exercism/problem-specifications#test-data-versioning) suggests for MAJOR.MINOR.PATCH that MINOR should be used when "Changing the test cases inputs and/or outputs." Whereas with "PATCH changes would never break well-designed test generators, because the test data remains exactly the same."

So while you'd assume that any robust matrix parser will handle a little extra whitespace without a glitch, the test data still does not remain exactly the same.